security/index.adoc

```diff
@@ -0,0 +1,86 @@
+[id="security-compliance-overview"]
+= {product-title} security and compliance
+include::modules/common-attributes.adoc[]
+:context: security-compliance-overview
+
+toc::[]
+
+// TODO: Need some help with an intro blurb
+
+[id="security-overview"]
+== Security overview
+
+It is important to understand how to properly secure various aspects of your {product-title} cluster.
+
+[discrete]
```

Comment:
Since anchors will still be autogenerated for these even though they're `[discrete]`, suggest adding unique IDs to them all.

Reply:
You know, I never really thought about that. And look at that, it's already in the guidelines with an example showing the discrete heading w/ an ID still. Thanks for pointing that out, I'll have to fix this in some of my other content sometime.